.github/PULL_REQUEST_TEMPLATE/dependency_change.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Dependency change
+about: Update one or more of the YoastCS dependencies
+labels: "yoast cs/qa"
+---
+
+## Description
+
+*
+
+Refs:
+<!-- Add relevant links to, for instance, the changelog of the updated dependency. -->
+*
+
+<!--
+If the dependency update involves PHP_CodeSniffer or the WordPress Coding Standards,
+please make sure that the `.travis.yml` script is also updated to reflect this change,
+so the unit tests are run against the relevant supported versions of these repos.
+-->
```

Comment:
I think test instructions could be relevant in this template as well: check whether this and this is done in the travis build etc. Having a test instructions heading in the template forces the PR writer to think about testing.

Reply:
The point of the remark about updating the Travis script, is that a dependency change on this repo does not need manual testing, but will automatically be tested via the Travis builds.

From another perspective: testing these type of changes for this repo isn't necessary in the first place.

All builds are tested against high/low versions of the current dependencies, with `high` being the current `develop` versions of the dependencies.

The only dependency changes which can be made in that sense, is the _raising_ of the minimum supported version. And that new minimum supported version has basically already been tested via Travis for a while.

Last point: the only exception to this would when it wouldn't be an "update" of one of the dependencies, but replacing a dependency or adding a new one, but that is not the situation targetted by this template.